app/frontend/src/features/messages/groupchats/CreateGroupChat.tsx

```diff
@@ -20,76 +19,67 @@ interface CreateGroupChatFormData {
 }
 
 export default function CreateGroupChat() {
-  const [loading, setLoading] = useState(false);
-  const [error, setError] = useState("");
-  const [friendIds, setFriendIds] = React.useState<number[]>([]);
-  const {
-    data: friends,
-    errors: queryErrors,
-    isLoading: queryIsLoading,
-  } = useUsers(friendIds);
-  const dispatch = useAppDispatch();
+  const classes = useStyles();
 
-  useEffect(() => {
-    (async () => {
-      setLoading(true);
-      try {
-        const friendIds = await service.api.listFriends();
-        setFriendIds(friendIds);
-      } catch (error) {
-        setError(error.message);
-      }
-      setLoading(false);
-    })();
-  }, [dispatch]);
+  const friends = useFriendList();
   const { control, register, handleSubmit } = useForm<
     CreateGroupChatFormData
   >();
 
-  const classes = useStyles();
+  const queryClient = useQueryClient();
+  const {
+    mutate: createGroupChat,
+    isLoading: isCreateLoading,
+    error: createError,
+  } = useMutation<number, GrpcError, CreateGroupChatFormData>(
+    (data) => service.conversations.createGroupChat(data.title, data.users),
```

Comment:
```suggestion
    ({ title, users ) => service.conversations.createGroupChat(title, users),
```
Probably nitpick tbh since it's only slightly shorter/terser in this case.